Intersect (Core)/Config/Options.cs

```diff
@@ -128,6 +128,8 @@ public class Options
 
         public static int RegenTime => Instance.CombatOpts.RegenTime;
 
+        public static bool CombatRegen => Instance.CombatOpts.CombatRegen;
```

Comment:
We're not doing new top level options like this anymore.
Please refer to the option through the category it is in.

Reply:
So would it be to remove the top-level option and then just call an:

` Options.Instance.CombatOpts.CombatRegen`

?

Reply:
Yeah, might be worth naming it something descriptive as well that tells the user what it does.